components/create_post/create_post.jsx

```diff
@@ -50,11 +51,6 @@ function trimRight(str) {
 export default class CreatePost extends React.Component {
     static propTypes = {
 
-        /**
-         *  ref passed from channelView for EmojiPickerOverlay
-         */
-        getChannelView: PropTypes.func,
```

Comment:
Is this related to this PR? It's good to clean it up here but might be better to deal with it separately as there's other place that needs to remove as well.

Reply:
Done.